mon-pix/app/components/head-content.js

```diff
@@ -0,0 +1,94 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { inject as service } from '@ember/service';
+import DS from 'ember-data';
+
+const PAGE_TITLE_MAP = {
+  'assessments.challenge': 'Evaluation en cours',
+  'assessments.checkpoint': 'Avancement du parcours',
+  'assessments.checkpoint.final': 'Fin de votre évaluation',
+  'campaigns.campaign-landing-page': 'Présentation | Parcours',
+  'campaigns.fill-in-campaign-code': 'Commencer un parcours',
+  'campaigns.skill-review': 'Résultat de votre parcours',
+  'campaigns.tutorial': 'Didacticiel | Parcours',
+  'certifications.start': 'Rejoindre une certification',
+  'competence-details': 'Compétence',
+  'competences.results': 'Résultat de votre compétence',
+  'inscription': 'Inscription',
+  'login': 'Connexion',
+  'password-reset-demand': 'Oubli de mot de passe',
+  'not-connected': 'Vous n\'êtes plus connecté',
+  'profile': 'Votre Profil',
+  'reset-password': 'Changer mon mot de passe',
+  'user-certifications.index': 'Mes certifications',
+};
+
+const CUSTOM_PAGES = [
+  'assessments.challenge',
+  'assessments.checkpoint',
+  'competence-details',
+  'competences.results',
+];
+
+const PAGE_TITLE_SUFFIX = ' | Pix';
+const DEFAULT_PAGE_TITLE = 'Pix – Mesurez, développez et valorisez vos compétences numériques';
+
+export default Component.extend({
+  router: service(),
+  store: service(),
+
+  pageTitle: computed('router.currentRouteName', function() {
+    return DS.PromiseObject.create({
+      promise: this._buildPageTitleName()
+    });
+  }),
+
+  async _buildPageTitleName() {
+    if (CUSTOM_PAGES.includes(this.router.currentRouteName)
```

Comment:
pour une lecture plus simple, je mettrais `this.router.currentRouteName` dans une variable vu son utilisation dedans.